### ISSUE TITLE
aop-prometheus is an invalid endpoint id

### ISSUE THREAD

**Issue by jasonmcintosh** [armory-plugins/armory-observability-plugin#19]
2020-08-21 16:11:25.000  WARN 1 --- [           main] o.s.boot.actuate.endpoint.EndpointId     : Endpoint ID 'aop-prometheus' contains invalid characters, please migrate to a valid format.

Should change the endpoint-id at some future release...

**Comment by EIrwin** [comment]
I am still see this as well in `clouddriver` logs.

**Comment by jasonmcintosh** [comment]
Yeah :) it's not a huge priority as it's a warning s. an error.  I figured sometime when I'd hit a 2.x release I'd look at changing the endpoint, as that'd be a breaking change.
